cli/pom.xml

```diff
@@ -145,7 +145,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <argLine>-Dspring.config.additional-location=optional:file://${user.home}/.l10n/config/cli/
-                        -Dspring.profiles.active=${user.name},test -Xmx1024m
+                        -Dspring.profiles.active=${user.name},test -Xmx2048m
```

Comment:
Is the memory heap increase for the tests still required?

Reply:
i don't think so, I added it when there was huge hibernate query generated and it was working only with mysql anyway I think. I've removed it and run a few test and it looked good